frontend-cert/css-projects/todo-list-stylized/user-stories.md

```diff
@@ -0,0 +1,21 @@
+1. You should have an `h1` element with the title `My To-Do List`.
+
+2. You should have an unordered list with the class `todo-list`.
+
+3. Inside the unordered list, you should have 4 list items. Each list item should have a checkbox, a task label and two links that point to a valid URL. 
+
+4. The first link in each list item should have the text `Edit` and the second link should have the text `Delete`. The links should also have the class `edit` and `delete` respectively.
+
+5. The list items should have a bottom margin of `15px` and a line height of `1.6`.
+
+6. Each list item should have squared bullets.
```

Comment:
They haven't learned about list style type here, so we should remove this requirement

Reply:
In the project sheet, there is a lecture with the title "How to work with the different list-style properties" just above this lab.
I assumed that some list style types would be taught there.
Should I still remove the `list-style-type` from the prototype?